protocols/interfaces.go

```diff
@@ -88,6 +88,11 @@ type Objective interface {
 // ObjectiveId is a unique identifier for an Objective.
 type ObjectiveId string
 
+// SortInfo needs to be implemented so that RejectionNotices can be received.
+func (id ObjectiveId) SortInfo() (channelID types.Destination, turnNum uint64) {
+	return
+}
```

Comment:
I'm a bit confused about this. Can you explain is some more?

Reply:
Does 8127ead437b133b8914a5f14b5c55eb3ea3710c5 clarify things?